test/nodes/test_preprocessor.py

```diff
@@ -15,15 +15,17 @@
 
 
 TEXT = """
-This is a sample sentence in paragraph_1. This is a sample sentence in paragraph_1. This is a sample sentence in
-paragraph_1. This is a sample sentence in paragraph_1. This is a sample sentence in paragraph_1.\f
 
-This is a sample sentence in paragraph_2. This is a sample sentence in paragraph_2. This is a sample sentence in
-paragraph_2. This is a sample sentence in paragraph_2. This is a sample sentence in paragraph_2.
 
-This is a sample sentence in paragraph_3. This is a sample sentence in paragraph_3. This is a sample sentence in
-paragraph_3. This is a sample sentence in paragraph_3. This is to trick the test with using an abbreviation\f like Dr.
-in the sentence.
+This is a sample sentence in paragraph_1 1. This is a sample sentence in paragraph_1 2. This is a sample sentence in
+paragraph_1 3. This is a sample sentence in paragraph_1 4. This is a sample sentence in paragraph_1 5.\f
+
+This is a sample sentence in paragraph_2 1. This is a sample sentence in paragraph_2 2. This is a sample sentence in
+paragraph_2 3. This is a sample sentence in paragraph_2 4. This is a sample sentence in paragraph_2 5.
+
+This is a sample sentence in paragraph_3 1. This is a sample sentence in paragraph_3 2. This is a sample sentence in
+paragraph_3 3. This is a sample sentence in paragraph_3 4. This is to trick the test with using an abbreviation\f like Dr.
+in the sentence 5 extra words.
 """
```

Comment:
I'm afraid this needs to be reverted: the duplicate sentences where there precisely to test that duplicates are not indexed. What lead you to this change? Can we work that around?